cli/command/container/completion.go

```diff
@@ -131,6 +131,7 @@ func addCompletions(cmd *cobra.Command, dockerCli command.Cli) {
 	_ = cmd.RegisterFlagCompletionFunc("sysctl", completion.NoComplete)
 	_ = cmd.RegisterFlagCompletionFunc("ulimit", completeUlimit)
 	_ = cmd.RegisterFlagCompletionFunc("user", completion.NoComplete)
+	_ = cmd.RegisterFlagCompletionFunc("userns", completion.FromList("host"))
```

Comment:
Note to self; looks like we don't define an enum/consts for this (there's only a single option, so maybe not really needed, but we do define a type for this; https://github.com/docker/cli/blob/32ff200fe6d843843718709ffa94023eb7356108/vendor/github.com/docker/docker/api/types/container/hostconfig.go#L171-L172